erpnext_com/www/docs/user/manual/en/setting-up/workflows.md

```diff
@@ -92,11 +101,12 @@ If a User is eligible to take action on some workflows, emails will be sent to t
 Also the users will see entries in their Workflow Action list.
 <img class="screenshot" alt="Workflow" src="{{docs_base_url}}/assets/img/setup/workflow-actions-list.png">
 
-**Note:** You can set email template for Workflow Actions on each state.
-The template might consist message for users to proceed with the next Workflow Actions
+**Note:**
 
+- You can set email template for Workflow Actions on each state. The template might consist message for users to proceed with the next Workflow Actions.
```

Comment:
The template might consist a message for users to proceed with the next Workflow Actions.